components/script/dom/element.rs

```diff
@@ -3632,19 +3632,21 @@ impl VirtualMethods for Element {
 
         let doc = document_from_node(self);
 
-        if let Some(ref shadow_root) = self.shadow_root() {
```

Comment:
This is no longer required because the ShadowRoot is now traversed by the code that calls unbind_from_tree.